docs/governance/fees/fees.md

```diff
@@ -25,3 +25,8 @@ The purpose of this page is to let users know which fee models are active at any
 Surplus is defined as the difference between the minimum execution price and the executed price of an order
 
 :::
+
+### Partner fee
+
+Partners might charge a partner fee when integrating with the protocol via the [Widget](/cow-protocol/tutorials/widget).
```

Comment:
This is true for any integration and not only widget (in fact we have two integrations pending for this partner fee that are custom integrations). I'd suggest to adjust accordingly: "Partners might charge a partner fee when integrating with the protocol."